cmd/config/config.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2020 Authors of Hubble
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+// New status command.
+func New(vp *viper.Viper) *cobra.Command {
+	configCmd := &cobra.Command{
+		Use:   "config",
+		Short: "Modify or view hubble config",
+		Long:  "Modify or view hubble config",
+		RunE: func(cmd *cobra.Command, _ []string) error {
+			return cmd.Help()
+		},
+	}
+	configCmd.AddCommand(
+		newSetCommand(vp),
+		newViewCommand(vp),
```

Comment:
Perhaps `hubble config get <KEY>` also?

Or meybe an optional `hubble config view <KEY>`.

Not blocking since easily mitigated via jq or grep.

Reply:
Added get command.